src-bindings/vscode/vscode.ml

```diff
@@ -2310,6 +2327,28 @@ module TreeItemCollapsibleState = struct
   [@@js.enum] [@@js]
 end
 
+module CustomDocument = struct
+  module type T = sig
+    include Js.T
+
+    val uri : t -> Uri.t
+
+    val dispose : t -> unit
+
+    val create : uri:Uri.t -> dispose:(unit -> unit) -> t
```

Comment:
I think the `create` function shouldn't be part of module type since modules that `include` this one might have more fields.

Reply:
Ah yes, that definitely makes sense. Thanks 👍

Reply:
Addressed in https://github.com/ocamllabs/vscode-ocaml-platform/pull/798/commits/446401d6471aabad92a3ff580e234ce41bc60def